src/pug/blog/year_one_recap.pug

```diff
@@ -0,0 +1,67 @@
+    li Our project got supported by #[em 10 wonderful folks in Patreon].
+    li We opened the #[em distribution channel (shop)] to better serve testers, reviewers, and everyone without the tools to build a controller from scratch. Setting up the logistics to build and deliver all these components has been a challenge, but also a success in verifying that the "DIY-or-buy" model for supporting open source projects can work for us.
+
+  h4 Exposure
+  ul
+    li We published #[+ax(org.youtube) 15 videos on Youtube], reaching a total of 58K views.
+    li The Alpakka controller was independently #[em reviewed by 5 Youtube channels].
+    li We were featured in the #[+ax('https://magpi.raspberrypi.com/issues/128') Raspberry Pi magazine], Heise magazine, and at least 10 other tech-related websites.
+    li The Alpakka controller was demoed among a few #[em videogame companies] with office in Helsinki, and had a small demo booth at the #[+ax('https://igda.org') IGDA] (International Game Developers Association) event.
+
+  h4 Development
+  ul
+    li There were #[em 21 firmware releases] (including betas), addressing lot of community requested features such as:
+    ul
+      li #[em Console Legacy] profile which makes easier to play emulators and retro games.
+      li #[em Racing] profile which allows to use the controllers as a steering wheel.
+      li #[em Generic] gamepad mode (aka DirectInput) implementation, which is useful both for retro gaming and for simulators expecting generic joysticks or steering wheels.
+      li Full #[em Steam Deck] compatibility.
+      li We created the #[em Ctrl app] as an alternative (and optional) method for configuring the Alpakka controller.
+    li There were #[em 5 3D-print models releases] with iterative improvements.
+    li There were #[em 3 PCB releases] with iterative improvements.
+
+  h4 Research
+  ul
+    li A total of #[+a('/db/stats') 44 games evaluated] for the accessibility game database.
+    li Published research notes about #[+a('gyro_ratcheting') Gyro ratcheting and flick-stick].
+    li Done in-depth research about #[+a('alphanumeric_input') Alphanumeric input methods] and implemented some of them in the Alpakka controller.
+
+  hr
+  h3 Things that could have gone better
+  ul
+    li We were too enthusiastic, and we #[em.wrap miscalculated the amount of work 2 individuals can achieve] even if working full-time. We got help from the community with the development, but there are many other aspects of running a company that can easily become a time sink (legal, taxes, insurances, accounting, logistics, support, etc).
+    li The #[em.wrap Kapybara controller development did not start yet], our original plan was to develop it alongside with the Alpakka, but eventually we realized that we do not have the resources to run 2 devices development in parallel, so we decided to delay the Kapybara development until we are done with wireless-related features on the Alpakka.
```

Comment:
Rephrase. Something like "We had hoped to put more effort into the Kapybara ..." "... but decided to prioritize the Alpakka for now."